src/fetchData.js

```diff
@@ -4,9 +4,42 @@ import { sortElements } from './sortElements.js'
  * @returns {object} cityDetails
  */
 export async function fetchCityDetails () {
-  let cityDetails = await fetch('../Assets/HTML&CSS/files/data.json')
+  let cityDetails = await fetch('https://soliton.glitch.me/all-timezone-cities')
     .then((res) => { return res.json() })
```

Comment:
Do we need .then blocks when await is used?

Reply:
no Sai, either  we can use either one...i will change it